packages/babel-types/src/definitions/core.ts

```diff
@@ -1838,7 +1838,8 @@ defineType("Super", {
 });
 
 defineType("TaggedTemplateExpression", {
-  visitor: ["tag", "quasi"],
+  visitor: ["tag", "quasi", "typeParameters"],
```

Comment:
I don't think we need this, since by default `visitor` should be `Object.keys(fields)`